pyproject.toml

```diff
@@ -1,7 +1,36 @@
-[tool.isort]
-profile = "black"
-lines_after_imports = 2
-src_paths = ["kinto-remote-settings", "tests"]
+[tool.ruff]
+# Exclude a variety of commonly ignored directories.
+exclude = [
+    ".git",
+    "__pycache__",
```

Comment:
It seems that this is the only rule that's not in the default excluded directories. Maybe we can use [`extend-exclude`](https://docs.astral.sh/ruff/settings/#extend-exclude) instead?